docs/content.zh/docs/connectors/table/hive/hive_read_write.md

```diff
@@ -163,6 +163,12 @@ following parameters in `TableConfig` (note that these parameters affect all sou
         <td>Integer</td>
         <td>Sets max infer parallelism for source operator.</td>
     </tr>
+    <tr>
+        <td><h5>table.exec.hive.load-partition-splits.thread-num</h5></td>
```

Comment:
This config option is not belong to `Source Parallelism Inference`, maybe we need to have a separate section, e.g. `Load Partiton Splits`

Reply:
@wuchong Sorry for the wrong place. I have modified it. Please review it again. Thanks.